source/Generic/PlayState/ViewModels/PlayStateManagerViewModel.cs

```diff
@@ -415,13 +430,23 @@ public bool SwitchGameState(PlayStateData gameData)
                         notificationType = processesSuspended ? NotificationTypes.Resumed : NotificationTypes.PlaytimeResumed;
                         gameData.Stopwatch.Stop();
                         logger.Debug($"Game {gameData.Game.Name} resumed in mode {gameData.SuspendMode}");
+                        if (isGameStatusOverrided)
+                        {
+                            gameData.IsGameStatusOverrided = true;
+                            gameData.GameStatusOverride = PlayStateDataStatus.Running;
+                        }
                     }
                     else
                     {
                         gameData.IsSuspended = true;
                         notificationType = processesSuspended ? NotificationTypes.Suspended : NotificationTypes.PlaytimeSuspended;
                         gameData.Stopwatch.Start();
                         logger.Debug($"Game {gameData.Game.Name} suspended in mode {gameData.SuspendMode}");
+                        if (isGameStatusOverrided)
+                        {
+                            gameData.IsGameStatusOverrided = true;
+                            gameData.GameStatusOverride = PlayStateDataStatus.Paused;
```

Comment:
I think it should use enums specific for this use case instead of the ones intended for something else

Reply:
Sure, I did that way to avoid adding more code but it's okay for me.

Reply:
Done